src/shogun/distributions/Gaussian.cpp

```diff
@@ -267,7 +267,7 @@ float64_t CGaussian::compute_log_PDF(SGVector<float64_t> point)
 	return -0.5 * answer;
 }
 
-SGVector<float64_t> CGaussian::get_mean()
+SGVector<float64_t>& CGaussian::get_mean()
```

Comment:
Hi @vigsterkr and @lisitsyn , the linalg add need a reference rather than a value like `add (SGVector< T > &a, SGVector< T > &b, SGVector< T > &result, T alpha=1, T beta=1)` , I just don't know how to make this 
`linalg::add( components[1]->get_mean(), components[2]->get_mean(),  components[1]->get_mean(), alpha1, alpha2);` works if I don't return a reference in here. Maybe I'm asking a dumb question, but I just don't know what should I do to make it works :(

Thank you